content/en/docs/distributions/ibm/deploy/install-kubeflow-on-iks.md

```diff
@@ -130,9 +133,116 @@ kfctl apply -V -f ${CONFIG_FILE}
+<hr/>
+
+### Using kustomize + kubectl
+
+1. Clone the manifest repo
+
+```
+git clone git@github.com:kubeflow/manifests.git
+```
+
+2. Checkout `v1.3-branch`.
+
+```
+git checkout v1.3-branch
+```
+
+3. Create a folder at the root folder. Create a `kustomiation.yaml` file in that folder.
+
+```
+mkdir ibm
+cd ibm
+touch kustomization.yaml
+```
+
+4. Populate the `kustomization.yaml` file with the following.
+
+```yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+# Cert-Manager (required)
+- ../distributions/stacks/ibm/application/cert-manager-kube-system-resources
+- ../distributions/stacks/ibm/application/cert-manager-crds
+- ../distributions/stacks/ibm/application/cert-manager
+# Istio (requires)
+- ../distributions/stacks/ibm/application/istio-1-9-0
+# OIDC Authservice (required)
+- ../distributions/stacks/ibm/application/oidc-authservice
+- ../distributions/stacks/ibm/application/dex-auth
+- ../distributions/stacks/ibm/application/user-namespace
+# KNative
+- ../distributions/stacks/ibm/application/knative
+# Kubeflow namespace (required)
+- ../distributions/stacks/ibm/application/kubeflow-namespace
+# Kubeflow Roles (required)
+- ../distributions/stacks/ibm/application/kubeflow-roles
+# Kubeflow istio resource (required)
+- ../distributions/stacks/ibm/application/kubeflow-istio-resources
+# Kubeflow Pipelines Argo
+# - ../distributions/stacks/ibm/application/kfp-argo
+# Kubeflow Pipelines Tekton
+- ../distributions/stacks/ibm/application/kfp-tekton
+# KFServing
+- ../distributions/stacks/ibm/application/kfserving
+# Katib
+- ../distributions/stacks/ibm/application/katib
+# Central Dashboard (required)
+- ../distributions/stacks/ibm/application/centraldashboard/single-user
+# Admission Webhook (required)
+- ../distributions/stacks/ibm/application/admission-webhook
+# Notebook Controller
+- ../distributions/stacks/ibm/application/notebook-controller
+# Jupyter Web App
+- ../distributions/stacks/ibm/application/jupyter-web-app/insecure
+# Profiles + KFAM (required)
+- ../distributions/stacks/ibm/application/profiles
+# Volumes Web App
+- ../distributions/stacks/ibm/application/volumes-web-app/insecure
+# Tensorboards Web App
+-  ../distributions/stacks/ibm/application/tensorboard-controller
+# Tensorboard Controller
+-  ../distributions/stacks/ibm/application/tensorboard-web-app/insecure
+# TFJob Operator
+- ../distributions/stacks/ibm/application/tf-job
+# Pytorch Operator
+- ../distributions/stacks/ibm/application/pytorch-job
+# MPI Operator
+- ../distributions/stacks/ibm/application/mpi-job
+# MXNet Operator
+- ../distributions/stacks/ibm/application/mxnet-job
+# XGBoost Operator
+- ../distributions/stacks/ibm/application/xgboost-job
+```
```

Comment:
I would hope we can keep these code in a git repo instead docs. I remembered it can be saved under stacks folder or distribution specific repo like kubeflow-dojo, it's going to be hard for users like me to copy&paste code from a doc

Reply:
I have the docs under `ibm/manifests` But until we have these merged back to `kubeflow/manifest` we would have to ask people to clone `ibm/manifests` instead. Which might be ok too. Asking for @Tomcli opinion

Reply:
Maybe we can have a page to teach how to use kustomize and what components are available for kustomize on Kubeflow. Then we can move these stuff to that new page.

Reply:
installing kustomize 3.2 for Kubeflow is also a difficult task for regular users especially when they have to downgrade their kustomize

Reply:
i have that documented.

Reply:
given there's a lot of redundancy on this kustomization.yaml, probably is a good idea to host it on a github repo.